frontend/components/sidebar/ViewSelector.tsx

```diff
@@ -25,43 +24,54 @@ const ViewButton = styled(Button)<ViewButtonProps>`
   }
 `;
 
-type ViewSelectorProps = Pick<AppState, 'setERView'>;
```

Comment:
Note: Pick is usable but there is an extra prop called show that needs to be added